aspnetcore/security/authentication/identity-api-authorization.md

```diff
@@ -177,6 +177,30 @@ services.Configure<JwtBearerOptions>(
     });
 ```
 
+The API's JWT handler raises events that enable control over the authentication process using `JwtBearerEvents`. To provide support for API authorization, `AddIdentityServerJwt` registers its own event handlers.
+
+To customize the handling of an event, wrap the existing event handler with additional logic as required. For example:
+
+```csharp
+services.Configure<JwtBearerOptions>(
+    IdentityServerJwtConstants.IdentityServerJwtBearerScheme,
+    options =>
+    {
+        var onTokenValidated = options.Events.OnTokenValidated;       
+        
+        options.Events.OnTokenValidated = async context =>
+        {
+            await onTokenValidated(context);
```

Comment:
`onTokenValidated` returns a `Task`, so we `await`. I'm calling this out as it's different from the snippet provided in the linked issue, so it might need to be verified accordingly.

I also went with `context` instead of `ctx` as that's what is usually used in the docs samples.

Reply:
Sounds good to me. I typed the code on gh, so expect it not to be perfect. I'm glad you caught those details :)